packages/react-scripts/template/src/registerServiceWorker.js

```diff
@@ -39,6 +60,8 @@ export default function register() {
           console.error('Error during service worker registration:', error);
         });
     });
+  } else {
+    renderToast('Development mode started.');
```

Comment:
Is the goal of this to ease users into the idea of the toast UI now being available? I haven't seen other implementations of the toast showing a dev mode message but perhaps it's another awareness moment.